tldraw/tldraw
# fix: proper label for opacity tooltip on hover

Describe what your pull request does. If appropriate, add GIFs or images showing the before and after.
There was a bug where hovering over the slider for opacity didn't resolve the tooltip properly, due to accessing object property incorrectly. This PR fixes this bug. The bug's reproduction video has been attached below.

Thank you for this project, grateful!

<details>
<summary>Bug Reproduction Video</summary>

https://github.com/tldraw/tldraw/assets/54077356/f313fe1e-6d83-43f7-b101-2818719649bd

</details>

### Change Type


[^1]: publishes a `patch` release, for devDependencies use `internal`
[^2]: will not publish a new version

### Test Plan

1. Open the app.
2. Hover above the opacity slider.
3. Ensure that the opacity value is shown properly.


### Release Notes

- Add a brief release note for your PR here.
